src/components/Users/ManualWebIdForm.jsx

```diff
@@ -0,0 +1,77 @@
+import React, { useState } from 'react';
```

Comment:
Not gonna add unit tests for this because we may be deleting it shortly. Keeping for demo purposes for now.